storage/SystemInfo/CmdParted.cc

```diff
@@ -39,7 +39,7 @@ namespace storage
     Parted::Parted(const string& device)
 	: device(device), label(PtType::PT_UNKNOWN), implicit(false), gpt_enlarge(false)
     {
-	SystemCmd cmd(PARTEDBIN " -s " + quote(device) + " unit cyl print unit s print", SystemCmd::DoThrow );
+	SystemCmd cmd(PARTEDBIN " -s " + quote(device) + " unit cyl print unit s print", SystemCmd::DoThrow);
```

Comment:
Please always use long form, so `--script` for `-s`. These are programs that need to be easy-to-read, no need to use shortcuts that are good only for typing faster. Thanks!